### ISSUE TITLE
When users type into search field, the screen reader fails to announce the characters being entered.:A11y_Azure Machine Learning Extension for VS Code_Settings_Screen reader

### ISSUE THREAD

**Issue by PoojaNamde** [Microsoft/vscode-tools-for-ai#2599]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #WCAG1.3.1; #Win32; #DesktopApp; #BM_Azure Machine Learning Extension for VS Code_Win32_Sept25; #Azure Machine Learning Extension for VS Code; #WCAG1.3.1; #FTP; #Screen Reader; #NVDA;

### Environment Details:
Application: Azure Machine Learning Extension for VS Code
Version: 1.2.1
OS: Windows 11 Enterprise 24H2 
OS build	26100.5074
NVDA Version: 2025.2
Visual studio code version: 1.103.2 (user setup)

### Repro steps:

1. Start NVDA/JAWS screen reader.
2. Install VS Code
3. Install the Azure Machine Learning extension and sign in with appropriate Credentials.
4. Open the Command Palette via the keyboard command Ctrl+Shift+P (or Command+Shift+P for MacOS)
5. Type "Open User Settings" and select the option from the dropdown list called "Preferences: Open User Settings") and press enter
6. In the search box, type "Azure Machine Learning".
7. Observe that When users type into search field, the screen reader fails to announce the characters being entered.

### Actual Result:
When users type into search field, the screen reader fails to announce the characters being entered.

**Issue does not repro with JAWS.**

### Expected Result:
As each character is typed into the search field, the screen reader should announce it immediately, allowing users to confirm their input in real time.

### User Impact:
Blind users rely on screen readers to confirm each character they type. Without auditory feedback, they cannot verify if they are typing correctly, leading to errors and frustration.

### Attachment:

https://github.com/user-attachments/assets/4b01afa0-da9c-4269-b545-8c4b78400609

https://github.com/user-attachments/assets/48f0de39-2a5f-4da1-9e3d-1da571d341a3

**Comment by PoojaNamde** [comment]
#Closed; #A11yFixed;

Closing this bug as no repro with NVDA.
